src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -264,16 +264,66 @@ def _update_progress_bars(self):
         '''
         clones = self._get_info_for_all_clones()
         if not clones:
-            self._finish()
+            self._wait_and_finish()
             return
 
         self._update_ongoing_progress_bar(clones)
         self._update_onpen_progress_bar(clones)
 
+    def _set_waiting_period_msg(self, time_left):
+        avg_progress_fraction = 1.0
+
+        msg_on = (f'All cloning finished, waiting {time_left} seconds before '
+                'exiting')
+        self._update_progress_bar_event(
```

Comment:
The code alread does that -- waiting for a period inactivity before terminating.

Reply:
IMO, upating progress module with this message is a a good idea since it informs users about this waiting. It also prevents us from running progress module objects, which can be reused if new clones are launched.

Reply:
Users have no interest in knowing that the internal timer is waiting for more newer cloner. It's an internal behaviour that's used to delay termination.

Reply:
The change has been made, marking this as resolved.